$NetBSD: patch-setup.cfg,v 1.1 2015/08/16 16:18:24 kleink Exp $

Do not include (dev) tag or date in egg name.

--- setup.cfg.orig	2015-03-26 15:49:54.000000000 +0000
+++ setup.cfg
@@ -1,6 +1,6 @@
 [egg_info]
-tag_build = dev
-tag_date = true
+tag_build =  
+tag_date = 0
 
 [aliases]
 release = egg_info -RDb ''
